addons/auth_signup/models/res_config_settings.py

```diff
@@ -1,24 +1,24 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from ast import literal_eval
-
-from odoo import api, fields, models
+from odoo import fields, models
 
 
 class ResConfigSettings(models.TransientModel):
     _inherit = 'res.config.settings'
 
-    auth_signup_reset_password = fields.Boolean(string='Enable password reset from Login page', config_parameter='auth_signup.reset_password')
-    auth_signup_uninvited = fields.Selection([
-        ('b2b', 'On invitation'),
-        ('b2c', 'Free sign up'),
-    ], string='Customer Account', default='b2b', config_parameter='auth_signup.invitation_scope')
-    auth_signup_template_user_id = fields.Many2one('res.users', string='Template user for new users created through signup',
-                                                   config_parameter='base.template_portal_user_id')
-
-    def open_template_user(self):
-        action = self.env["ir.actions.actions"]._for_xml_id("base.action_res_users")
-        action['res_id'] = literal_eval(self.env['ir.config_parameter'].sudo().get_param('base.template_portal_user_id', 'False'))
-        action['views'] = [[self.env.ref('base.view_users_form').id, 'form']]
-        return action
+    auth_signup_reset_password = fields.Boolean(
```

Comment:
Lot of unnecessary diff in this file, makes real changes hard to understand. Could you maybe limit to real changes ?

Reply:
Well, those files are often messy and chunk of codes are copy/pasted everywhere because people seldom understand how it is supposed to work. 

As I made a round on all the files, I took the opportunity to make them clean. 

Concerning the 1 line by attribute, this is quite subjective but it allows to limit the modifications of 1 attribute to 1 line (and not the whole one) to those fields that have more field attributes in general than in common odoo models, and actually limit the forward port conflicts (once you passed this commit, I agree with you).

This will be quite time-consuming to revert only those changes, so I'll keep it that way.